pkgs/development/python-modules/graphlib-backport/default.nix

```diff
@@ -0,0 +1,40 @@
+{ lib
+, fetchFromGitHub
+, buildPythonPackage
+, setuptools
+, poetry-core
+}:
+
+buildPythonPackage {
+  pname = "graphlib-backport";
+  version = "1.0.3";
+
+  format = "pyproject";
+
+  src = fetchFromGitHub {
+    owner = "mariushelf";
+    repo = "graphlib_backport";
+    rev = "113473ecf2b5b296cadcd1d2439908dca8affc01"; # No tags in repo
```

Comment:
a tag is available for v1.1.0: https://github.com/mariushelf/graphlib_backport/releases/tag/1.1.0

Reply:
Updated!